common/app/views/support/cleaner/AmpEmbedCleaner.scala

```diff
@@ -61,22 +61,29 @@ case class AmpEmbedCleaner(article: Article) extends HtmlCleaner {
   }
 
   def cleanAmpEmbed(document: Document) = {
+    document.getElementsByClass("element-embed")
+      .filter(_.getElementsByTag("iframe").length != 0)
+      .foreach(_.getElementsByTag("iframe").map(_.remove))
+  }
```

Comment:
Nice simplification! You could probably remove some nesting from `cleanAmpInstagram` by removing one or two arrow functions in the same way.